script/package.ts

```diff
@@ -1,16 +1,27 @@
-#!/usr/bin/env node
-
-'use strict'
-
-const fs = require('fs-extra')
-const cp = require('child_process')
-const path = require('path')
-const distInfo = require('./dist-info')
-
-const distPath = distInfo.getDistPath()
-const productName = distInfo.getProductName()
+/* tslint:disable:no-sync-functions */
+
+import * as fs from 'fs-extra'
+import * as cp from 'child_process'
+import * as path from 'path'
+import {
+  getDistRoot,
+  getDistPath,
+  getProductName,
+  getOSXZipPath,
+  getWindowsIdentifierName,
+  getCompanyName,
+  getWindowsStandaloneName,
+  getWindowsInstallerName,
+  shouldMakeDelta,
+  getUpdatesURL,
+} from './dist-info'
+
+const distPath = getDistPath()
+const productName = getProductName()
 const outputDir = path.join(distPath, '..', 'installer')
 
+import * as electronInstaller from 'electron-winstaller'
```

Comment:
Could we move this up to be with the other imports?

Reply:
Also, this won't fail at runtime on non-Windows platforms will it?

Reply:
> Also, this won't fail at runtime on non-Windows platforms will it?

Dunno, I'm pretty sure you can install the module on non-Windows platforms, but the scripts are setup so they don't execute anything in the module. Will test.

Reply:
Tested on all the platforms - `yarn run package` works as intended.